esy-lib/Git.ml

```diff
@@ -73,7 +79,7 @@ let lsRemote ?ref ~remote () =
     | Some ref -> Cmd.(cmd % ref)
     | None -> cmd
   in
-  let%bind out = ChildProcess.runOut cmd in
```

Comment:
The changes to this file are refactoring to allow `lsRemote` to share the same `runGit` method - since it now returns a string, the other places had to be updated to return unit